tests/dummy/app/components/chart-highstock-interactive.js

```diff
@@ -18,7 +20,7 @@ export default class BarBasic extends Component {
     }
   };
 
-  chartData = copy(stockData, true);
+  chartData = clone(stockData);
```

Comment:
@Gaurav0 I know this is an old PR, but is there any chance there is anything `structuredClone` doesn't support in this chart data? I know it won't copy functions for example.

Reply:
While many things don't work with structuredClone, it is faster and better than ember-copy in what it does support.

Reply:
Sure, but is there anything we will break by making the switch?

Reply:
Anything that relies on it implementing the Copyable mixin.

Reply:
Keep in mind that functions, DOM nodes, etc. cannot really be "cloned" in JavaScript. ember-copy creates a reference in the new object to the original, which "silently works" in some cases, but breaks when you actually switch JavaScript contexts. IMO, an error is better. And chart data shouldn't contain these anyway. However, it is up to you to decide whether this is a "bugfix" or a "breaking change". My 2 cents is document it and call it a bugfix.

Reply:
I'm fine with that. Just wanted to double check. Thanks!